src/zarr/storage/local.py

```diff
@@ -143,17 +149,6 @@ async def get_partial_values(
         prototype: BufferPrototype,
         key_ranges: Iterable[tuple[str, ByteRangeRequest]],
     ) -> list[Buffer | None]:
-        """
-        Read byte ranges from multiple keys.
```

Comment:
Did you mean to delete all these docstrings?

Reply:
Yes I did :)

These are now inherited from the base class. The only reason to redefine them is if extra context or notes on specific behavior is needed.